src/app/custom/translations.tsv

```diff
@@ -960,6 +948,11 @@
 "geography"	"Geography"	"Géographie"
 "classification"	"Classification"	"Classification"
 "indexation"	"Indexation"	"Indexation"
+"homogeneisation"  "Homogeneisation"  "Homogénéisation"
+"validation"  "Validation"  "Validation"
+"entites nommees"  "Named entities"  "Entités nommées"
+"pretraitement"  "Preprocessing"  "Prétraitement"
+"tal"  "Automatic language processing"  "Traitement automatique de la langue"
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** Tabulation in TSV, not spaces
‏
```suggestion
"homogeneisation"	"Homogeneisation"	"Homogénéisation"
"validation"	"Validation"	"Validation"
"entites nommees"	"Named entities"	"Entités nommées"
"pretraitement"	"Preprocessing"	"Prétraitement"
"tal"	"Natural language processing"	"Traitement automatique de la langue"
```

Reply:
To incorporate my suggestions to the file, use the `Commit suggestion` button:
![image](https://github.com/user-attachments/assets/5d049b3f-6f33-4556-85ac-0db65b9f268d)